src/lib/slack/callSlackHook.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'node-fetch';
+import debug from 'debug';
+import envConfig from '../../envConfig';
+
+const dlog = debug('that:api:sessions:slack-callslackhook');
+
+export default function callSlackHook(hookBody) {
+  dlog('calling Slack hook');
+  if (
+    process.env.NODE_ENV === 'production' ||
+    process.env.TEST_SLACK_NOTIFICATIONS
+  ) {
+    const slackUrl = envConfig.slackWebhookUrl;
+    fetch(slackUrl, {
+      method: 'post',
+      body: JSON.stringify(hookBody),
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .then(res => res.text())
+      .then(res => dlog('Slack webhood response: %o', res))
```

Comment:
did you want to return the result from the function? cause it's being swallowed up by the next .then.

Reply:
Only returning the result during dev, not need response. Perhaps should send an error to sentry other than nowhere. Wasn't to concerned if this failed.

Reply:
Sentry seems like a good idea.

Reply:
Corrected in commit 761deac